### PR TITLE
Update Overnight Sessions.md

### PR DESCRIPTION
**Pull request by r6lm**
Temporarily, change the night closure time to 8 pm.

### CODE REVIEW

File: Posit Infrastructure/Overnight Sessions.md [hunk 1]
```diff
@@ -6,13 +6,13 @@ This document aims to provide users with information on running Posit Workbench
 
 ## Background
 
-An automated process has been put in place to close sessions in Posit Workbench at 9pm every evening.
+An automated process has been put in place to close sessions in Posit Workbench at 8pm every evening.
```
Comment (terrymclaughlin) [inline Posit Infrastructure/Overnight Sessions.md:9]: Could we leave this line as is, but add in a new sentence e.g.

`_During Daylight Saving Time, sessions will close at 8pm._`

File: Posit Infrastructure/Overnight Sessions.md [hunk 2]
```diff
@@ -6,13 +6,13 @@ This document aims to provide users with information on running Posit Workbench
...
 
 This automated process can be overridden by including the word "NIGHT" at the start of the name of the relevant Posit Workbench session, **when the session is started**.
 
 ## Process to start a "NIGHT" session
 
-A Posit Workbench session that a user would like to continue running past 9pm must be named with the word "NIGHT" **when the session is started**.  The following steps must be carried out to ensure that the session remains open past 9pm:
+A Posit Workbench session that a user would like to continue running past 8pm must be named with the word "NIGHT" **when the session is started**.  The following steps must be carried out to ensure that the session remains open past 8pm:
```
Comment (terrymclaughlin) [inline Posit Infrastructure/Overnight Sessions.md:15]: As per previous comment.

File: Posit Infrastructure/Overnight Sessions.md [hunk 3]
```diff
@@ -26,4 +26,4 @@ A Posit Workbench session that a user would like to continue running past 9pm mu
 
 4. Amend any other settings as required in the "New Session" dialog box, and click the blue "Start Session" button.  The session will start as normal.
 
-**Important Note:** If you open a project from within a session started by following the steps above, the process of opening that project will start another session and close your "NIGHT" session.  As such, the session that the project opens in will no longer have "NIGHT" at the start of its name, and it will be closed at 9pm by the automated process.
+**Important Note:** If you open a project from within a session started by following the steps above, the process of opening that project will start another session and close your "NIGHT" session.  As such, the session that the project opens in will no longer have "NIGHT" at the start of its name, and it will be closed at 8pm by the automated process.
```
Comment (terrymclaughlin) [inline Posit Infrastructure/Overnight Sessions.md:29]: As per previous comment.

**Review by terrymclaughlin — CHANGES_REQUESTED** [review]
Please see comments about Daylight Saving Time.

**Comment by r6lm** [comment]
I have acted upon your comment, @terrymclaughlin . I believe just one sentence is enough.

File: Posit Infrastructure/Overnight Sessions.md [hunk 4]
```diff
@@ -6,13 +6,13 @@ This document aims to provide users with information on running Posit Workbench
 
 ## Background
 
-An automated process has been put in place to close sessions in Posit Workbench at 9pm every evening.
+An automated process has been put in place to close sessions in Posit Workbench every evening. During **Daylight Saving Time**, sessions will close at **8pm**.
```
Comment (Moohan) [inline Posit Infrastructure/Overnight Sessions.md:9]: The non DST time was dropped.
```suggestion
An automated process has been put in place to close sessions in Posit Workbench every evening at 9pm. During **Daylight Saving Time**, sessions will close at **8pm**.
```

Comment (r6lm) [reply Posit Infrastructure/Overnight Sessions.md:9]: I don't see much value in keeping the 9 pm reference in the meantime. Less is more.

**Review by terrymclaughlin — APPROVED** [review]

**Pull request merged**